library/spdm_requester_lib/libspdm_req_get_measurements.c

```diff
@@ -609,6 +601,11 @@ static libspdm_return_t libspdm_try_get_measurement(libspdm_context_t *spdm_cont
     #endif /* LIBSPDM_ENABLE_MSG_LOG */
 
 receive_done:
+    if ((status != LIBSPDM_STATUS_SUCCESS) &&
+        (status != LIBSPDM_STATUS_NOT_READY_PEER) &&
+        (status != LIBSPDM_STATUS_BUSY_PEER)) {
```

Comment:
`LIBSPDM_STATUS_BUSY_PEER` needs to be clarified by the SPDM Working Group. Right now the specification says that only `LIBSPDM_STATUS_NOT_READY_PEER` is applicable here. See https://github.com/DMTF/SPDM-WG/issues/2917.

Reply:
OK. I dropped BUSY.